src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Callbacks;
+
+namespace Sentry.Unity.Editor.Native
+{
+    public static class BuildPostProcess
+    {
+        [PostProcessBuild(1)]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
+        {
+            if (target is not (BuildTarget.StandaloneWindows or BuildTarget.StandaloneWindows64))
+            {
+                return;
+            }
+
+            var options = ScriptableSentryUnityOptions.LoadSentryUnityOptions(BuildPipeline.isBuildingPlayer);
+            var logger = options?.DiagnosticLogger ?? new UnityLogger(new SentryUnityOptions());
+
+            try
+            {
+                if (options?.Validate() != true)
+                {
+                    logger.LogWarning("Failed to validate Sentry Options. Native support disabled.");
+                    return;
+                }
+
+                if (!options.WindowsNativeSupportEnabled)
+                {
+                    logger.LogDebug("Windows Native support disabled through the options.");
+                    return;
+                }
+
+                var crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins",
+                    "Windows", "Sentry", "crashpad_handler.exe"));
+                var targetPath = Path.Combine(Path.GetDirectoryName(pathToProject)!, Path.GetFileName(crashpadPath));
```

Comment:
`Path.GetDirectoryName(pathToProject)!` when can this method return null?
Should we assign to a variable and check for null and log/bail instead of `!` in here?

Reply:
actually, it cannot be null. fixed